Formula/w/wgpu-native.rb

```diff
@@ -0,0 +1,39 @@
+class WgpuNative < Formula
+  desc "Native WebGPU implementation based on wgpu-core"
+  homepage "https://github.com/gfx-rs/wgpu-native"
+  url "https://github.com/gfx-rs/wgpu-native.git",
+    tag:      "v25.0.2.1",
+    revision: "af9074edf144efe4f1432b2e42c477429c4964c1"
+  license "Apache-2.0"
+  head "https://github.com/gfx-rs/wgpu-native.git", branch: "trunk"
+
+  depends_on "rust" => :build
+  depends_on "cmake" => :test
+  depends_on "ninja" => :test
+
+  def install
+    # Not using `cargo install` because wgpu-native doesn't ship binaries
+    system "cargo", "build", "--release", "--locked"
+
+    # Manually install artifacts
+    include.install "ffi/webgpu-headers/webgpu.h"
+    include.install "ffi/wgpu.h"
+    (include/"webgpu-headers").install_symlink "../webgpu.h" => "webgpu.h"
+
+    lib.install "target/release/#{shared_library("libwgpu_native")}"
+    lib.install "target/release/libwgpu_native.a"
+
+    # Install examples for sake of testing
+    pkgshare.install "examples"
+  end
+
+  test do
+    system "cmake", "-G", "Ninja",
+           "-S", pkgshare / "examples",
```

Comment:
```suggestion
           "-S", pkgshare/"examples",
```
We omit the spaces around `/` for `Pathname`s.